easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.1.2-iccifort-2018.1.163-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.1.2'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2018.1.163-GCC-6.4.0-2.28'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e5bf7b2d7517f5cb7c9e1fed15d2df2da501669072477b21a513e892a22663bc']
+
+dependencies = [('hwloc', '1.11.7')]
```

Comment:
@edmondac This should have been bumped to `1.11.8`, since the existing `OpenMPI-2.1.2-GCC-6.4.0-2.28.eb` (part of `foss/2018a`) that merged months before this one already depended on `hwloc` 1.11.8, I overlooked this.

I propose we rectify this via #5994, so we can maintain a "one dependency variant" policy starting with the `*/2018a` (common) toolchains.